Uni_life_tracker/app/src/main/res/layout/activity_expenses_landing.xml

```diff
@@ -0,0 +1,132 @@
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.149"
+            app:layout_constraintHorizontal_bias="0.172"
+            android:fontFamily="@font/roboto_mono_bold"
+            android:textSize="36sp" android:textColor="#070707"
+            android:id="@+id/expenses_title"/>
+
+
+    <TextView
+            android:layout_width="270dp"
+            android:layout_height="96dp"
+            android:text="@string/welcome_please_add_an_expense"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            android:layout_marginBottom="400dp"
+            android:id="@+id/e_welcome_text"
+            android:fontFamily="@font/robotomono"
+            android:textColor="#070707" android:textSize="18sp"
+            android:gravity="center" tools:ignore="HardcodedText"/>
+
+    <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:id="@+id/e_horizontal_line"
+            tools:layout_editor_absoluteY="-85dp"
+            tools:layout_editor_absoluteX="-16dp">
+
+
+        <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/e_add_name"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:errorEnabled="true">
+        </com.google.android.material.textfield.TextInputLayout>
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginTop="250dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_name"
+                android:inputType="text"
+        />
+        <com.google.android.material.textfield.TextInputLayout
+                android:id="@+id/e_add_amount"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintRight_toRightOf="parent"
+                app:errorEnabled="true">
+        </com.google.android.material.textfield.TextInputLayout>
+
+
+        <com.google.android.material.textfield.TextInputEditText
+                android:layout_width="240dp"
+                android:layout_marginStart="95dp"
+                android:layout_height="wrap_content"
+                android:hint="@string/add_amount"
+                android:inputType="text"
```

Comment:
I think the inputType should be a number instead of text, I'm pretty sure there is a different attribute for that

Reply:
Good Catch